packages/mobile/src/components/app-navigator/BottomTabNavigator.tsx

```diff
@@ -71,6 +72,7 @@ const createStackScreen = <StackParamList extends ParamListBase>(
       <Stack.Screen name='EditProfile' component={EditProfileScreen} />
       <Stack.Group>
         <Stack.Screen name='FollowersScreen' component={FollowersScreen} />
+        <Stack.Screen name='FavoritedScreen' component={FavoritedScreen} />
```

Comment:
Heads up, just merged in a change that moves this into `BaseStackNavigator.tsx`

Reply:
great, thanks updating